guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -151,7 +151,7 @@ Use the following procedure to create an activation key that you can use to subs
 . Select your new *CentOS Stream* product in the table.
 . Click *Add Selected* to save.
 
-Use your activation key to register a host to the Content View. For more information on registering a host, see {ManagingHostsDocUR}registering-a-host-to-project-using-the-global-registration-template_managing-hosts
+Use your activation key to register a host to the Content View. For more information on registering a host, see {ManagingHostsDocURL}registering-a-host_managing-hosts
```

Comment:
This passed HTML validator because there was a typo and variable was not recognized, so it did not start as an URL.

Reply:
Interesting!